static/src/javascripts/projects/commercial/modules/third-party-tags/imr-worldwide.spec.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 import { imrWorldwide } from './imr-worldwide';
 
-const { shouldRun, url, onLoad } = imrWorldwide;
+const { shouldRun, url } = imrWorldwide;
+const onLoad: any = imrWorldwide.onLoad;
```

Comment:
Yeah it's a good question. I don't mind using `any` for tests - quite often we're using simplified versions of things for readability, and as long as the test is easy to understand I think good flow discipline isn't as important here.